docs/src/testing/advanced/fuzz-testing.md

```diff
@@ -0,0 +1,59 @@
+# Fuzz Testing
+
+In many cases, a test needs to verify function behavior for multiple possible values.
+While it is possible to come up with these cases on your own, it is often impractical, especially when you want to test
+against a large number of possible arguments.
+
+> ℹ️**Info**
+> Currently, Forge only supports using randomly generated values for fuzzing.
+> This way of fuzzing doesn't support any kind of value generation based on code analysis, test coverage or results of
+> other fuzzer runs.
+> In the future, more advanced fuzzing execution modes will be added.
+
+## Random Fuzzing
+
+> ⚠️ **Warning**
+> Currently Forge only supports fuzz testing arguments with `felt252` type. Trying to use different argument types will
```

Comment:
There was a section in the docs about the type of test functions. It should be adjusted in this PR